app/src/main/java/io/intrepid/contest/splash/SplashActivity.java

```diff
@@ -0,0 +1,51 @@
+package io.intrepid.contest.splash;
+
+import android.os.Bundle;
+import android.os.PersistableBundle;
+import android.support.annotation.NonNull;
+
+import butterknife.OnClick;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseMvpActivity;
+import io.intrepid.contest.base.PresenterConfiguration;
+import timber.log.Timber;
+
+
+public class SplashActivity extends BaseMvpActivity<SplashContract.Presenter> implements SplashContract.View {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState, PersistableBundle persistentState) {
+        super.onCreate(savedInstanceState, persistentState);
+    }
+
+    @NonNull
+    @Override
+    public SplashContract.Presenter createPresenter(PresenterConfiguration configuration) {
+        return new SplashPresenter(this, configuration);
+    }
+
+    @Override
+    protected int getLayoutResourceId() {
+        return R.layout.activity_splash;
+    }
+
+    @OnClick({ R.id.create_contest_button })
+    public void onCreateButtonClick() {
```

Comment:
It would be nice to be consistent with the verb tense (onCreateButtonClick vs onJoinContestClicked). I'm using the past tense but I can change it if you decide to go for the present tense.